src/cli/command-workspace.ts

```diff
@@ -31,6 +33,31 @@ Then:
 echo 'PASTE TOKEN HEX HERE' > ./github-token
 `;
 
+/**
+ * Post-Conversion steps that the user can select to run after workspace
+ * conversion.
+ */
+enum PostConversionStep {
+  Test = 'test',
```

Comment:
I'd suggest slightly different prompts:

- Install dependencies and run tests
- Install dependencies only
- Exit